api/src/prescription/campaign/infrastructure/serializers/csv/campaign-assessment-export.js

```diff
@@ -224,6 +241,19 @@ class CampaignAssessmentExport {
 
     return acquiredBadgesByCampaignParticipations;
   }
+
+  async #getAcquiredStagesByCampaignParticipations({ campaignParticipationInfoChunk, stageAcquisitionRepository }) {
+    let acquiredStagesByCampaignParticipations = null;
+
+    const campaignParticipationsIds = campaignParticipationInfoChunk.map((info) => info.campaignParticipationId);
+
+    if (this.stageCollection.hasStage) {
```

Comment:
suggestion: faire un early return en haut de la fonction si !this.stageCollection.hasStage